easybuild/easyconfigs/r/RELION/RELION-1.4_fltk-config.patch

```diff
@@ -0,0 +1,11 @@
+--- configure.orig	2014-08-11 17:18:31.686813247 +0200
++++ configure	2014-08-11 17:18:39.390957858 +0200
+@@ -15665,7 +15665,7 @@
+ #Check for FLTK
+ # as g++ seems needed and not gcc for compilation,
+ #just check for presence of fltk-config program
+-fltkfile=`pwd`/bin/fltk-config
++fltkfile=`which fltk-config`
```

Comment:
You might want to use `command -v` instead of `which`: the former is guaranteed to be a shell built-in by POSIX, the latter is a BSD addition usually implemented by an "external" command.